app/src/main/java/org/mozilla/fenix/compose/list/ExpandableListHeader.kt

```diff
@@ -76,7 +76,7 @@ fun ExpandableListHeader(
 
                 Icon(
                     painter = painterResource(
-                        if (expanded) R.drawable.ic_chevron_down else R.drawable.ic_chevron_up
+                        if (expanded) R.drawable.ic_chevron_up else R.drawable.ic_chevron_down
```

Comment:
![image](https://user-images.githubusercontent.com/1190888/164341755-72a65c7f-70d3-49d0-88ee-bb851a4532f6.png)

Not sure what to make of this right now since collections actually does the opposite of what the component library is proposing. Will check with @cwzilla tomorrow.

Reply:
You're probably right though when I look at https://material.io/components/lists#specs. Again will check with @cwzilla tomorrow. I would generally recommend moving these types of fixes into a separate PR and having them land immediately since the changes are easy to verify and small in nature so very easy to review and just land.